ext/storage_sources/gcp_store/gcp_storage_source.cpp

```diff
@@ -58,8 +59,11 @@ static int gcp_remove(WT_FILE_SYSTEM *, WT_SESSION *, const char *, uint32_t)
   __attribute__((__unused__));
 static int gcp_rename(WT_FILE_SYSTEM *, WT_SESSION *, const char *, const char *, uint32_t)
   __attribute__((__unused__));
-static int gcp_file_size(WT_FILE_SYSTEM *, WT_SESSION *, const char *, wt_off_t *)
+static int gcp_file_lock(WT_FILE_HANDLE *, WT_SESSION *, bool) __attribute__((__unused__));
+static int gcp_file_size(WT_FILE_HANDLE *, WT_SESSION *, wt_off_t *) __attribute__((__unused__));
```

Comment:
Just a genuine question, are these `__attribute__((__unused__))` here on purpose?

Reply:
yup, they are used to prevent warnings as some functions have been defined but not used, they will be removed altogether as part of [WT-10425](https://jira.mongodb.org/browse/WT-10425)

Reply:
Sounds good, thanks!